keyboardvisibilityevent/build.gradle

```diff
@@ -21,6 +21,7 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.50"
+    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
```

Comment:
[lifecycle-extensions is deprecated.](https://developer.android.com/jetpack/androidx/releases/lifecycle#declaring_dependencies)

use `androidx.lifecycle:lifecycle-runtime` instead.

Reply:
Replaced